js/speakerdata.js

```diff
@@ -0,0 +1,69 @@
+
+
+
+const speakersData = [
+
+    {
+        id: '1',
+        name: 'Burnaboy',
+        span: 'gbedu',
+        description: ' a song writer a producer a performer 10 times grammy aaward winner and bet award and the king of afro pop',
+        featured_image2: '../images/Burna.png',
```

Comment:
- [ ] Please write the name of the image correct as it should be like that `featured_image2: '../images/burna.png',`.